src/test/java/com/google/firebase/auth/FirebaseAuthIT.java

```diff
@@ -140,37 +141,98 @@ public void testDeleteNonExistingUser() throws Exception {
   @Test
   public void testCreateUserWithParams() throws Exception {
     RandomUser randomUser = RandomUser.create();
-    String phone = randomPhoneNumber();
-    CreateRequest user = new CreateRequest()
-        .setUid(randomUser.uid)
-        .setEmail(randomUser.email)
-        .setPhoneNumber(phone)
-        .setDisplayName("Random User")
-        .setPhotoUrl("https://example.com/photo.png")
-        .setEmailVerified(true)
-        .setPassword("password");
-
-    UserRecord userRecord = auth.createUserAsync(user).get();
+    String randomPhoneNumber = randomPhoneNumber();
     try {
-      assertEquals(randomUser.uid, userRecord.getUid());
-      assertEquals("Random User", userRecord.getDisplayName());
-      assertEquals(randomUser.email, userRecord.getEmail());
-      assertEquals(phone, userRecord.getPhoneNumber());
-      assertEquals("https://example.com/photo.png", userRecord.getPhotoUrl());
-      assertTrue(userRecord.isEmailVerified());
-      assertFalse(userRecord.isDisabled());
-
-      assertEquals(2, userRecord.getProviderData().length);
+      UserRecord user = createUser(randomUser.uid, randomPhoneNumber, randomUser.email);
+      assertEquals(randomUser.uid, user.getUid());
+      assertEquals("Random User", user.getDisplayName());
+      assertEquals(randomUser.email, user.getEmail());
+      assertEquals(randomPhoneNumber, user.getPhoneNumber());
+      assertEquals("https://example.com/photo.png", user.getPhotoUrl());
+      assertTrue(user.isEmailVerified());
+      assertFalse(user.isDisabled());
+
+      assertEquals(2, user.getProviderData().length);
       List<String> providers = new ArrayList<>();
-      for (UserInfo provider : userRecord.getProviderData()) {
+      for (UserInfo provider : user.getProviderData()) {
         providers.add(provider.getProviderId());
       }
       assertTrue(providers.contains("password"));
       assertTrue(providers.contains("phone"));
 
-      checkRecreate(randomUser.uid);
+      checkRecreate(user.getUid());
     } finally {
-      auth.deleteUserAsync(userRecord.getUid()).get();
+      auth.deleteUserAsync(randomUser.uid).get();
+    }
+  }
+
+  @Test
+  public void testLookupUserByPhone() throws Exception {
+    RandomUser randomUser1 = RandomUser.create();
+    String phoneNumber1 = null;
```

Comment:
nit: dead code. Either remove it or use it on line 179. (Personally, I'd lean toward removing it.)